source/chapters/advanced_topics.rst

```diff
@@ -106,6 +106,2734 @@ send messages to text displays on the controller.
 For more information, go to `<https://mixxx.org/wiki/doku.php/midi_scripting>`_
 and `<https://mixxx.org/wiki/doku.php/hid_mapping_format>`_.
 
+Mixxx Controls
+--------------
+
+See :ref:`controlindex` for a full list.
+
+ControlPotMeter controls
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+
+The following extensions add some features to ControlPotMeter controls
+(volume, crossfader, ...). Use in conjunction with ``[Channel*N*]``,
+``[Sampler*N*]``, ``[Master]`` ... groups. Please note, this doesn't work in
+JavaScript files, so you can not do, for example:
+
+.. code-block:: javascript
+
+   engine.setValue('MyController', 'keylock_toggle')
```

Comment:
I made the note more clear on the wiki, but I don't have time to copy it over here and apply formatting for a proper suggestion.. :\
for detail please see https://github.com/mixxxdj/mixxx/wiki/Mixxxcontrols/_compare/fcef2c24516d49f7dd8746868db5931bb5b5cfd9...fdf8a2fad2558a43fe4341913802ec130d7b8c3c